types/evse_manager.yaml

```diff
@@ -236,24 +236,38 @@ types:
     enum:
       - Car
       - CarDiodeFault
-      - Relais
       - RCD
       - RCDDC
       - VentilationNotAvailable
-      - OverCurrent
-      - OverVoltage
-      - UnderVoltage
       - Internal
       - SLAC
       - HLC
       - Other
-      - MotorLock
-      - OverTemperature
       - BrownOut
-      - CablePP
       - EnergyManagement
-      - NeutralPEN
-      - CpDriver
+      - MREC_1_ConnectorLockFailure
+      - MREC_2_GroundFailure
+      - MREC_3_HighTemperature
+      - MREC_4_OverCurrentFailure
+      - MREC_5_OverVoltage
+      - MREC_6_UnderVoltage
+      - MREC_8_EmergencyStop
+      - MREC_10_InvalidVehicleMode
+      - MREC_11_CableCheckFailure
+      - MREC_12_PreChargeFailure
+      - MREC_14_PilotFault
+      - MREC_15_PowerLoss
+      - MREC_16_EVContactorFault
+      - MREC_17_EVSEContactorFault
+      - MREC_18_CableOverTempDerate
+      - MREC_19_CableOverTempStop
+      - MREC_20_PartialInsertion
+      - MREC_21_CapacitanceFault
+      - MREC_22_ResistanceFault
+      - MREC_23_ProximityFault
```

Comment:
Do we need to differentiate between:
- ProximityFault (out of range proximity voltage)
- No cable detected during charing e.g. NoCableDuringCharging

Reply:
I would emit the ProximityFault in both cases (Trying to add as few custom error codes as possible, this maximizes the use of harmonized error codes)